elastic/kibana
# [Controls] fix time slider play button display=fill property not being respected

resolves https://github.com/elastic/kibana/issues/145301

Steps to test
* create new dashboard
* Add time slider control
* verify play toggle button has primary color background

<img width="600" alt="Screen Shot 2022-11-16 at 9 56 00 AM" src="https://user-images.githubusercontent.com/373691/202244344-e4ec6f45-9d87-4819-96a6-dc47b551b213.png">
